src/vmm/src/config.rs

```diff
@@ -23,6 +94,42 @@ pub enum ConversionError {
     ParseBlock(String),
 }
 
+#[derive(Debug, PartialEq)]
+enum ParsingError {
+    /// Missing required argument
+    MissingRequired(String),
+    /// Parsing failed, param and error.
+    ParsingFailed(&'static str, String),
+}
+
+impl ToString for ParsingError {
+    fn to_string(&self) -> String {
+        match self {
+            Self::MissingRequired(err) => format!("Missing required argument: {}", err),
+            Self::ParsingFailed(param, err) => {
+                format!("Param '{}', parsing failed: {}", param, err)
+            }
+        }
+    }
+}
```

Comment:
As per [this recommendation](https://doc.rust-lang.org/std/string/trait.ToString.html) will it not be better to `impl Display` for this ?